src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="position">
+    <form
+      action="#"
+      data-qa="big"
+      class="form form__big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        class="big__input"
+        placeholder="Try “Los Angeles“"
```

Comment:
The class name 'big__input' suggests a style or size which is not ideal. Class names should describe the content's meaning and not its appearance.